frontend/public/components/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -87,8 +87,8 @@ export const DetailsCard_ = connect(mapStateToProps)(({
           {openshiftFlag ? (
             <>
               <DetailItem
-                key="name"
-                title="Name"
+                key="clusterid"
+                title="Cluster ID"
                 value={getClusterName()}
```

Comment:
By cluster ID, I mean the actual UID of the cluster from the `ClusterVersion` resource. I think this is more useful to admins since it's what you would use in cluster manager. (See the cluster settings page or about dialog.)

Reply:
aah, ok. Should be ok now.